pages/reference/OS/updates/update-process.md

```diff
@@ -3,32 +3,108 @@ title: Update process details
+
+__Note:__ If you've used Advanced HUP, it does not mean necessarily updates won't be compatible with Standard HUP. For instance, if a device running 32-bit  {{ $names.os.lower }} on a Raspberry Pi 3 has been updated to a 64-bit variant of {{ $names.os.lower }}, Standard HUP can be used for future updates to the 64-bit OS. In this case, Advanced HUP is only used once to migrate the device from one OS architecture to another.
+
+### Caveats
+When using the Advanced HUP, it’s important to be aware of several limitations compared to the Standard HUP mechanism.
+
+#### Data loss
+Data loss may occur, as the process can involve overwriting partitions and filesystems. Please make sure to backup all data on the device before running any update. Also, after a successful update, all application containers will be downloaded again.
+
+#### No automatic rollbacks
+Advanced HUP does not provide a built-in rollback mechanism. If the new OS version proves unstable or introduces unexpected issues, there is no automated way to revert to a previous state. 
+
+#### No delta updates
+The OS image is downloaded directly via the ap. The registry and {{ $names.engine.lower }} is not used during the download process.
+
+### How it works
+Similar to Standard HUP, an updater script gets transferred to the device to handle the update process.
+
+The first step the updater script performs includes a number of coherence checks, so that an update does not progress if something isn't right. The checks include, among other things, whether the device is running a {{ $names.os.lower }} version that it can work with, whether the script is running on the right device type, and whether it can reach the {{ $names.cloud.lower }} api to get the target {{ $names.os.lower }} image.
+
+Next, the updater script downloads the required OS image from the api. This differs from Standard HUP whereby the OS image is downloaded from the registry and is the reason why Advanced HUP does not provide delta updates. The updater script then downloads the [takeover binary][takeover] which is responsible for the main update process.
+
+The updater script hands over to `takeover` which:
+- kills running processes
+- copies required files and artifacts to RAM
+- Unmounts partitions
+- Flashes the target OS image to disk already running {{ $names.os.lower }}
+- Transfers required files that were copied to RAM to their respective destinations on the newly flashed OS
+
+The device is then rebooted.
+
+You can read more about the tool [here](https://github.com/balena-os/takeover?tab=readme-ov-file#how-it-works).
+
+__Note:__ Because the disk running {{ $names.os.lower }} is overwritten with the target OS image, there is no automated rollback mechanism when using Advanced HUP. However, if the process is interrupted before the disk is overwritten, the device is rebooted and it will boot into the OS that was already running.
```

Comment:
Not ideal to start a note and a new sentence with `because`

```suggestion
Since the disk running {{ $names.os.lower }} is overwritten with the target OS image during advanced HUP, there is no automated rollback mechanism. However, if the process was interrupted before the disk was overwritten, the device reboots into the OS that was already running.
```